adapters/zwave/zwave-classifier.js

```diff
@@ -46,7 +47,28 @@ class ZWaveClassifier {
             'name': 'toggle',
             'description': 'Toggles On/Off',
         }];
-        node.propertyMap = {'on': valueId };
+        node.propertyMaps = {'on': valueId };
+        node.values.on = node.zwValues[valueId].value;
+
+        if (node.zwInfo.productId == 0x0060 &&
+            (node.zwInfo.productType & 0xff) == 0x03) {
```

Comment:
Not that I'm aware of. The reporting behaviour of the switch is controlled by a config which defaults to "Don't report anything when the switch is pressed", which allows the state of the light to become out of sync.

I won't know how prevelant this will be until we experiment with different brands of modules. My Leviton switches, for example, don't change the light and don't report anything when the button is pressed (so at least its consistent).

From what research I did, there is no standard for having config options, and what config options exist seem to be entirely vendor specific.

So, we need someway of specifying this in order to get "sensible" behaviour. Obviously, it doesn't need to exist buried in the code like this. You could have some type of external file which has quirks. For that matter, you could probably even have thing mappings be described purely using data.